taskana-adapter-camunda-outbox-rest-spring-boot/src/main/java/pro/taskana/adapter/camunda/outbox/rest/springboot/OutboxRestServiceSpringBootApplication.java

```diff
@@ -0,0 +1,12 @@
+package pro.taskana.adapter.camunda.outbox.rest.springboot;
```

Comment:
Please remove the OutboxRestServiceSpringBootApplication class, because the outbox-rest-spring-boot project should proved a library that can be included into another spring boot application...